controller/src/beerocks/master/tasks/client_steering_task.cpp

```diff
@@ -249,11 +249,18 @@ void client_steering_task::steer_sta()
 void client_steering_task::handle_event(int event_type, void *obj)
 {
     if (event_type == STA_CONNECTED) {
-        TASK_LOG(DEBUG) << "steering successful for sta " << sta_mac;
-        steering_success = true;
+        auto current_bssid = database.get_node_parent(sta_mac);
```

Comment:
This is confusing. I would like it more if it was "newly_connected_bssid'. (nitpick)